lib/psulib_blacklight/solr_config.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+require 'zip'
+
+module PsulibBlacklight
+  class SolrConfig
+    CONFIG_PATH = '/solr/admin/configs'
+    COLLECTION_PATH = '/solr/admin/collections'
```

Comment:
@cdmo I guess you keep them here as part of config but since these are only used by solr_manager what about moving these to solr_manager.rb?

Reply:
Semantics is the only reason there's another class really. SolrManager is the only thing that calls SolrConfig. Borrowing from the SS4 works, I think it's only two classes because it's too much for one class. So the more straight-config type stuff goes in SolrConfig, where the Manager is more the actions taken upon collections and configs. That's how I think about it. What do you think?

Reply:
Yeah, I figured your thoughts on this and see your point, I am not picky about this, was just pointing it. What I am thinking is that these two won't get changed and would be a bit shorter when used in the manager class just like

`list = connection.get(CONFIG_PATH, action: 'LIST')`